pacts/http/serlo.org-commenting_system.json

```diff
@@ -44,7 +44,7 @@
               {
                 "id": "ce118b6e-d8e1-11e7-9296-cec278b6b50a",
                 "content": "Ich habe folgende Frage",
-                "author": {
+                "user": {
```

Comment:
That should not be changed here but rather in the consumer, i.e. serlo.org.

Reply:
So for this PR: just note that we (intentionally) broke the API and need to adjust the consumers